pybind11/style.rst

```diff
@@ -270,13 +254,21 @@ This is especially again common in :ref:`declare functions <style-guide-pybind11
 Organization
 ============
 
+.. _style-guide-pybind11-wrap-split-modules:
+
+Wrappers for a header SHALL be implemented in functions prefixed with "wrap"
+----------------------------------------------------------------------------
+
+Wrappers for ``LinearTransform`` (declared in ``LinearTransform.h``, with wrappers in ``_LinearTransform.cc``) should go in a function called ``wrapLinearTransform``.
```

Comment:
"SHALL" and then "should"?  Sounds wishy-washy.  Also "Typically" below — what are the exceptions?

Reply:
Agree the language is inconsistent, though I might prefer to downgrade the `SHALL` to `SHOULD` - for both that and "typically" below, I feel like we shouldn't make a not-very-important naming rule a very strict one unless we're quite certain there's no valid exception we should defer to developer judgement.  I know we have a general statement about that in style guide introduction, but I don't think that's a substitute for leaving room for and emphasizing the importance of developer judgement throughout.

I especially don't want to wall ourselves in too much until we've tried to apply these guidelines to afw.